compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -45,15 +45,7 @@ impl<'tcx> MirPass<'tcx> for Inline {
             return enabled;
         }
 
-        match sess.mir_opt_level() {
-            0 | 1 => false,
-            2 => {
-                (sess.opts.optimize == OptLevel::Default
-                    || sess.opts.optimize == OptLevel::Aggressive)
-                    && sess.opts.incremental == None
-            }
-            _ => true,
-        }
+        sess.mir_opt_level() > 0
```

Comment:
We need to keep the incremental opt-out that existed for opt-level=2. So I think all that should be done here is to move the `| 1` to the second match arm.